packages/abi-gen/package.json

```diff
@@ -19,7 +19,7 @@
         "test:coverage": "nyc npm run test --all && yarn coverage:report:lcov",
         "coverage:report:lcov": "nyc report --reporter=text-lcov > coverage/lcov.info",
         "test_cli": "run-p test_cli:test_typescript diff_contract_wrappers",
-        "test_cli:clean": "rm -rf test-cli/output && rm -rf test-cli/test_typescript/lib",
+        "test_cli:clean": "rm -rf test-cli/test_typescript/lib",
         "test_cli:build": "tsc --project test-cli/tsconfig.json",
```

Comment:
This was bothering me because we're cleaning up generated code upon `yarn clean`, which i think is a good idea, but we're also checking in that generated code, so it pollutes the `git status` with missing files.  And I don't think we want to .gitignore those files, right? (If we just ignore them, why even have them checked in?)

Reply:
Yeah, good call! This has been bothering me too, thanks for fixing it.